charts/flyte-core/values.yaml

```diff
@@ -277,12 +277,6 @@ datacatalog:
 
 flyteagent:
   enabled: false
-  plugin_config:
-    plugins:
-      agentService:
-        defaultAgent:
-          endpoint: "dns:///flyteagent.flyte.svc.cluster.local:8000"
-          insecure: true
```

Comment:
Are we able to add `supportedTaskTypes` to this section? since we should only add this config when agent is enabled.

Reply:
Yeah! I can restructure the templates to do the right thing.